### PR TITLE
Various minor tweaks

### PR DESCRIPTION
**Pull request by jrfnl**
## Proposed Changes

Just a couple of commits with minor tweaks for things I noticed along the way.

### CS: minor tweaks

.. for improved code readability and consistency.

### CONTRIBUTING: fix code of conduct link

Only just noticed that it hadn't been adjusted for the move of the CoC to a separate file.

### CODE REVIEW

**Review by Potherca — APPROVED** [review]

File: phpcs.xml.dist
Comment (jrfnl) [reply phpcs.xml.dist:24]: Generally speaking, there are two schools of thought about these types of rules:
* **Pro**: it makes code more readable.
* **Con**: it makes `git blame` more involved/less reliable (like you said).

In most projects, code is _read_ a lot more often and by a lot more people, than that the history of a particular change is traced back,
With that in mind, a lot of projects choose to let _code readability_ win over _code history_.

I agree with that the rule has drawbacks as you outlined. Then again, the same can be said about certain other things.

An example in this code base is that there are no array formatting rules which are being enforced. This means that a trailing comma at the end of an array entry isn't being enforced, which in turn means that if someone forgot to add one, the array entry above a new entry would also change when a new array entry is being added. This codebase already had one instance of such a missing trailing comma - see [line 309 in the plugin code](https://github.com/Dealerdirect/phpcodesniffer-composer-installer/pull/151/files#diff-c346b1aeac8259782c74702eadfe1d7a240ee3ecea6020ad2addede93c635cabR309). 

Once the tests get added, the code for the plugin itself will be only a small subset of the code in the codebase, with the test code being the majority.
As I noticed I was applying these rules (assignment operator alignment, trailing comma's in arrays, array assignment operator alignment) when creating the test code, it made sense to me to try and enforce them for consistency.

Unfortunately, there aren't suitable array related sniffs in PHPCS itself, so for the array rules, this will need to wait until PHPCSExtra releases the full [`NormalizedArrays` ruleset](https://github.com/phpcsstandards/phpcsextra#normalizedarrays), which we could then configure and add.

We could also choose to just not enforce these rules (not add it to the ruleset) and just leave it up to the individual contributor, but that will make the codebase less consistent in the long run.

Either way, these were my considerations for adding the rule.

Comment (Potherca) [reply phpcs.xml.dist:24]: I think the "makes code more readable" is also a personal matter. For me the added spaces for alignment make it less readable but I am aware I am a minority there... And my dev setup is well-behaved enough not to make it bothersome.

For now, lets just leave it in. Like you said, there is currently a general lack of enforcement.
This might be a good starting point. :+1:

**Pull request merged**
